pkgs/applications/misc/kaufkauflist/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, buildGoModule, fetchFromGitHub, esbuild, buildNpmPackage, fetchFromGitea }:
+
+let
+  esbuild' = esbuild.override {
+    buildGoModule = args: buildGoModule (args // rec {
+      version = "0.16.15";
+      src = fetchFromGitHub {
+        owner = "evanw";
+        repo = "esbuild";
+        rev = "v${version}";
+        hash = "sha256-iTAtPHjrBvHweSIiAbkkbBLgjF3v68jipJEzc0I4G04=";
+      };
+      vendorSha256 = "sha256-+BfxCyg0KkDQpHt/wycy/8CTG6YBA/VJvJFhhzUnSiQ=";
+    });
+  };
+in buildNpmPackage rec {
+  pname = "kaufkauflist";
+  version = "1.0.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "annaaurora";
+    repo = "kaufkauflist";
+    rev = "v${version}";
+    hash = "sha256-H+ss0l85AeAM2MfhXqElEZwcLaTvaXqloLc2gueF4oM=";
+  };
+
+  npmDepsHash = "sha256-lSnGLK7+ac/wEpAxlpkZS/kgr9F+8WK+nRjCzkrPJt0=";
+
+  ESBUILD_BINARY_PATH = "${lib.getExe esbuild'}";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/kaufkauflist $out/share/pocketbase
+    cp -vr build/* $out/share/kaufkauflist/
+    cp -v pb_schema.json $out/share/pocketbase/
```

Comment:
I asked this earlier (don't think you responded to it), but does it make sense to have the database schema served alongside the site itself? I guess it really doesn't matter, but thought I'd ask.

Reply:
You said that I should expose the database schema somewhere and @artemislena who wants to use this package agrees. The difference to not including it isn't that big since you can't automate importing the schema yet. Lets just leave it like this.